lib/sources/__tests__/github-resource-source.js

```diff
@@ -102,26 +102,26 @@ describe('GithubResourceSource', async() => {
 
             expect(() => {
                 const source = new GithubResourceSource(
-                    logger, 
-                    client, 
-                    axclient, 
-                    {                       
+                    logger,
+                    client,
+                    axclient,
+                    {
                     });
                 }).toThrowError("You must supply a repository URL");
         });
 
     })
 
-    describe('begin', async() => {
+    describe('begin', () => {
```

Comment:
Same comment as the resource loader. Why the empty tests?

Reply:
Same cause.  I've added some minimal tests against the possibility of future changes to functionality.